ultra/ultra/scenarios/generate_scenarios.py

```diff
@@ -832,6 +838,11 @@ def build_scenarios(
     parser.add_argument(
         "--root-dir", help="directory of maps", type=str, default="ultra/scenarios"
     )
+    parser.add_argument(
+        "--no-mission-shuffle",
+        help="Do not shuffle ego missions.",
+        action="store_false",
+    )
```

Comment:
Do we need a parser inside of the main() in `generate_scenarios.py`? I believe the CLI arguments are taken into affect by the `interface.py`. I have not utilize the `generate_scenarios.py` for generating scenarios from the command line

Reply:
I think we can create a new branch and run ours tests after removing the parser (I guess we might not even a main method) and see if any errors occurred. I am quite sure that there wouldn't be any issue